src/common/components/elements/Button2.tsx

```diff
@@ -12,6 +12,16 @@ const BUTTON_BASE_CLASSES = {
   text: '',
 };
 
+const BUTTON_DISABLED_CLASSES = {
```

Comment:
This can actually be achieved with the disabled: keyword in front of the class :) I already made it in the "final" centralized button in the storybook branch, HolaButton (working title)